src/common/paths/write.yml

```diff
@@ -208,8 +208,9 @@ post:
         Bad request. The response body contains detail about the error.
 
         InfluxDB returns this error if the line protocol data in the request is malformed.
-        The response body contains the first malformed line in the data.
-        InfluxDB rejected the batch and did not write any data.
+        The response body contains the first malformed line in the data, and indicates what was expected.
```

Comment:
@sunbryely-influxdata, not sure if indicates what was expected is always true. If it's not, we could drop that, or say in some cases.

Reply:
ill do a bit more testing to see if we should drop that line

Reply:
thank you, @sunbryely-influxdata! cc @Joe-Influx